apps/server/src/modules/authentication/controllers/dto/login.response.ts

```diff
@@ -1,10 +1,14 @@
-import { ApiProperty } from '@nestjs/swagger';
+import { ApiProperty, ApiPropertyOptional } from '@nestjs/swagger';
 
 export class LoginResponse {
```

Comment:
Maybe we could also make a dedicated OauthLoginResponse response, so that ldap and local don't have the externalIdToken attribute in their api declaration.

Reply:
Good idea. done